implementations/rust/ockam/ockam_core/src/flow_control/flow_controls/flow_controls_debug.rs

```diff
@@ -1,33 +0,0 @@
-use crate::flow_control::FlowControls;
-use crate::Address;
-
-impl FlowControls {
-    /// Prints debug information regarding Flow Control for the provided address
-    #[allow(dead_code)]
-    pub fn debug_address(&self, address: &Address) {
```

Comment:
It's still used by `access_control.rs`, `debugger` feature should fail

Reply:
I would really appreciate it if you could keep some form of extra logging, so far, it really helped me track the flow in complex situations

Reply:
@davide-baldo I added some logs when you add producers/spawners/consumers. I hoped it would be enough to debug if something didn't pass the access control. E.g., if you see the "did not pass outgoing access control" message, you should already see the Addresses. And then you can use those Addresses to search the log and see how it was setup?